Rakefile

```diff
@@ -14,24 +14,39 @@ RDoc::Task.new(:rdoc) do |rdoc|
   rdoc.rdoc_files.include('lib/**/*.rb')
 end
 
-APP_RAKEFILE = File.expand_path("../test/dummy/Rakefile", __FILE__)
+APP_RAKEFILE = File.expand_path("../test/rails_apps/devise/Rakefile", __FILE__)
```

Comment:
In order to maintain consistency with other spots the devise app is still the default.